packages/transport/src/modules/messages/MessageController.ts

```diff
@@ -297,12 +297,8 @@ export class MessageController extends TransportController {
         const deletedPeers: string[] = [];
         const peersWithMissingOrInactiveRelationship: string[] = [];
         for (const recipient of parsedParams.recipients) {
-            const relationship = await this.relationships.getRelationshipToIdentity(recipient);
-            if (
-                relationship &&
-                (relationship.status === RelationshipStatus.Terminated || relationship.status === RelationshipStatus.Active) &&
-                relationship.peerDeletionInfo?.deletionStatus !== "Deleted"
-            ) {
+            const relationship = await this.relationships.getActiveOrTerminatedRelationshipToIdentity(recipient);
+            if (relationship && relationship.peerDeletionInfo?.deletionStatus !== "Deleted") {
```

Comment:
I think you can make it even simpler. I suggest
- get Relationship without specifying the status
- check if peer is deleted, if so push to `deletedPeers` and `continue`
- check if Relationship doesn't exist, if so push to `peersWithMissingOrInactiveRelationship` and `continue`
- otherwise process for usual case (without the need for `if` or `else`)

Doing so, you don't need a new function `getActiveOrTerminatedRelationshipToIdentity.